apps/src/storage/firebaseStorage.js

```diff
@@ -1241,9 +1241,13 @@ FirebaseStorage.getColumn = function (
     tableName,
     {},
     records => {
-      let columnValues = [];
-      records.forEach(row => columnValues.push(row[columnName]));
-      onSuccess(columnValues);
+      if (records === null) {
+        onSuccess(null);
+      } else {
+        let columnValues = [];
+        records.forEach(row => columnValues.push(row[columnName]));
```

Comment:
I think this can just be a map.

```
const columnValues = records.map(row => row[columnName])
```

Reply:
Oh, also just as another note in case anyone looks to this code again...

even if it's not switched to the map, it should still be `const columnValues = []`. It doesn't need to be a `let`.

`const` only prevents you from re-assigning the variable itself (e.g., `columnValues = 'foo'` later on wouldn't be allowed), but you can still manipulate the object itself. JavaScript is weird. 🤷

Reply:
I really appreciate this comment and will keep it in mind going forward. Because this is in the critical path for the AP CSP Create task though, I want to keep the change as scoped as possible and only touch the "records is null" fork of the code path. So I'll leave it as is for this PR. I would follow-up and change it after Create task next month, but since this file will be deleted in a couple months anyway, I'm going to skip the overhead. BUT I will remember this fact about const, and how much cleaner map is than the forEach push song and dance! Thank you!!